docs/reference/contrib/frontendcache.md

```diff
@@ -105,10 +105,6 @@ WAGTAILFRONTENDCACHE = {
 }
 ```
 
-```{versionchanged} 6.2
-Previous versions allowed passing a dict for `DISTRIBUTION_ID` to allow specifying different distribution IDs for different hostnames. This is now deprecated; instead, multiple distribution IDs should be defined as [multiple backends](frontendcache_multiple_backends), with a `HOSTNAMES` parameter to define the hostnames associated with each one.
-```
```

Comment:
Added a commit to remove this. I think this was the only remaining one of the deprecation notes.